src/number_theory/primorial.lean

```diff
@@ -5,24 +5,29 @@ open_locale big_operators
 open finset
 open nat
 
+def primorial (n : ℕ) : ℕ := ∏ p in (filter prime (range (n + 1))), p
+notation x`#` := primorial x
+
 -- This is going into stdlib
 lemma sum_range_choose_halfway (m : ℕ) : ∑ i in range (m + 1), choose (2 * m + 1) i = 4 ^ m :=
    by sorry
 
 private lemma choose_symm_half (m : ℕ) : choose (2 * m + 1) (m + 1) = choose (2 * m + 1) m :=
 by apply choose_symm_of_eq_add; ring
 
-lemma extend_prime_product_odd (n : ℕ) (n_big : 2 < n) (r : n % 2 = 0) :
-  ∏ p in (filter prime (range n)), p = ∏ p in (filter prime (range (n + 1))), p
-  :=
+lemma primorial_succ (n : ℕ) (n_big : 2 < n) (r : n % 2 = 1) : n# = (n + 1)# :=
```

Comment:
Is this the direction you usually want to rewrite in? My intuition would expect the left hand side and right hand side of the statement to be swapped.

Reply:
Yeah, I just swapped that round locally five minutes ago